### PR TITLE
Eggscellent additions

### PR DESCRIPTION
**Pull request by LordBarkBread**
#### Summary
Content "Omelets"

#### Purpose of change
Omelets are a simple, tasty, and a wonderful meal in the real world and I noticed that while we have scrambled and fried eggs, we do not have the true breakfast of champions.

#### Describe the solution
Add in comestibles and recipe for **three** new eggy additions, omelet, vegetable omelet, and supreme omelet. 

#### Describe alternatives you've considered
Not cracking a few eggs.

#### Testing
N/A

#### Additional context
I thought about adding in omelet sandwiches or in general 'breakfast' sandwiches, but for now all I'm concerned about is the tasty omelet.

### CODE REVIEW

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -5231,6 +5231,66 @@
     ],
     "charges": 2
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "omelete",
```
Comment (TheShadowFerret) [inline data/json/recipes/recipe_food.json:5237]:
```suggestion
    "result": "omelet",
```

File: data/json/recipes/recipe_food.json [hunk 2]
```diff
@@ -5231,6 +5231,66 @@
...
+    "components": [
+      [ [ "eggs_bird", 2, "LIST" ], [ "egg_reptile", 2 ] ],
+      [ [ "cheese_standard", 1, "LIST" ] ],
+      [ [ "any_butter_or_oil", 1, "LIST" ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "vegetable_omelete",
```
Comment (TheShadowFerret) [inline data/json/recipes/recipe_food.json:5256]:
```suggestion
    "result": "vegetable_omelet",
```

File: data/json/recipes/recipe_food.json [hunk 3]
```diff
@@ -5231,6 +5231,66 @@
...
+      [ [ "eggs_bird", 2, "LIST" ], [ "egg_reptile", 2 ] ],
+      [ [ "cheese_standard", 1, "LIST" ] ],
+      [ [ "veggy_any", 2, "LIST" ] ],
+      [ [ "any_butter_or_oil", 1, "LIST" ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "supreme_omelete",
```
Comment (TheShadowFerret) [inline data/json/recipes/recipe_food.json:5276]:
```suggestion
    "result": "supreme_omelet",
```

**Review by TheShadowFerret — COMMENTED** [review]
~~I thought the only way to spell omelette was the British way before this~~

**Comment by LordBarkBread** [comment]
> ~I thought the only way to spell omelette was the British way before this~

I had a bit of a back and forth in cata discord and with my husband over how do we spell omelet while I was doing this and...man it shows that my mind was boggled so badly. xD

**Comment by tenmillimaster** [comment]
EGG

**Comment by akrieger** [comment]
Is this... ready?

**Comment by LordBarkBread** [comment]
> Is this ready? It looks ready but I don't know if there's more polishing in the works.

It is! I had been running around trying to figure out what the issues were on my own. And I fixed them! :>

**Pull request merged**